bot/handlers/administration/create_key_handler.py

```diff
@@ -0,0 +1,40 @@
+from typing import (
+    Awaitable,
+    Callable,
+    List,
+)
+
+from aiogram.types import Message
+
+from bot.database.database_manager import DatabaseManager
+from bot.handlers.bot_message_handler import BotMessageHandler
+from bot.responses.administration.create_key_handler_responses import (
+    get_create_key_success_message,
+    get_create_key_usage_message,
+)
+
+
+class CreateKeyHandler(BotMessageHandler):
+    def get_commands(self) -> List[str]:
+        return ["addkey", "addk"]
+
+    def _get_validator_functions(self) -> List[Callable[[Message], Awaitable[bool]]]:
+        return [
+            self._validate_argument_count,
+        ]
+
+    @staticmethod
+    async def _validate_argument_count(message: Message) -> bool:
+        args = message.text.split()
+        if len(args) < 3:
+            await message.answer(get_create_key_usage_message())
+            return False
+        return True
```

Comment:
przejrzyj wszedzie gdzie to robisz i uzyj tej ujednoliconej funkcji, juz nie bede robic 5 takich samych komow